packages/record/src/index.ts

```diff
@@ -6,4 +7,5 @@ export default {
   TrialRecordExtension,
   StartRecordPlugin,
   StopRecordPlugin,
+  ConsentVideoPlugin,
```

Comment:
How annoying would it be to change this to `VideoConsentPlugin` (for consistency with EFP's `video-consent` frame and my `VideoConfigPlugin`)?

Reply:
Can do!